From a90c578952219e740f24d9f2560f54f19ae7e906 Mon Sep 17 00:00:00 2001
From: Philip Withnall <withnall@endlessm.com>
Date: Wed, 11 Apr 2018 15:45:10 +0100
Subject: [PATCH] tests: Lower number of iterations in network-monitor-race

While 333 runs is very likely to reproduce the bug, Milan has previously
reproduced it with as few as 9 runs. Since this test will be run by the
CI machinery quite often, a lower number of runs each CI run will still
probably catch any regressions over time.

This reduces the total test runtime from 33s to 2s.

https://bugzilla.gnome.org/show_bug.cgi?id=793727

Signed-off-by: Philip Withnall <withnall@endlessm.com>
Reviewed-by: nobody
---
 gio/tests/network-monitor-race.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio/tests/network-monitor-race.c b/gio/tests/network-monitor-race.c
index 4b92c87a5..cadd62cee 100644
--- a/gio/tests/network-monitor-race.c
+++ b/gio/tests/network-monitor-race.c
@@ -18,7 +18,7 @@
 #include <glib/glib.h>
 #include <gio/gio.h>
 
-#define MAX_RUNS 333
+#define MAX_RUNS 20
 
 static gboolean
 quit_loop (gpointer user_data)
-- 
2.17.0

